requirements/quality-extra.txt

```diff
@@ -0,0 +1,2 @@
+pylint
+edx-lint                  # edX pylint rules and plugins
```

Comment:
I think the rationale for having it included is to have repeatable builds (it only gets updated when you explicitly run `pip-compile` or `make upgrade`).  But if the library is imported in an external project, I'm not sure that goal is really acheived, since it will install the version from setup.py, not from base.txt.